src/app/routes/utils/regex/index.test.js

```diff
@@ -550,7 +550,7 @@ describe('frontPage -> homePage migration', () => {
     .filter(service => !servicesWithVariants.includes(service))
     .map(serviceToRoute);
 
-  const migratedServices = ['kyrgyz'];
+  const migratedServices = ['kyrgyz', 'azeri'];
```

Comment:
**nitpick**: Can we put these in alphabetical order?

Reply:
This is an excellent nitpick, good idea :)